packages/block-editor/src/components/inserter/hooks/use-block-types-state.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import { map } from 'lodash';
```

Comment:
You can just use the vanilla [`[].map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map), which even IE 9 supports!

Reply:
This was just copy/pasted as is and I don't think it's guaranteed that innerBlocks there is always an array (it can be undefined).

Reply:
Then use optional chaining? But if you want to leave it as is to keep the number of changes in this PR low, that's fine. :+1: